app/lib/actions/katello/content_view/export.rb

```diff
@@ -0,0 +1,58 @@
+module Actions
+  module Katello
+    module ContentView
+      class Export < Actions::EntryAction
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        ISO_OUTPUT_BASEDIR = "/var/lib/pulp/published/yum/master/group_export_distributor/"
+
+        def plan(content_view, export_to_iso, since, iso_size)
+          action_subject(content_view)
+          content_view.check_ready_to_publish!
+          start_date = since ? since.iso8601 : nil
+
+          sequence do
+            repo_pulp_ids = content_view.repositories_to_publish.collect { |r| r.pulp_id }
```

Comment:
(continued from previous comment)  content_view.repositories is just the list of repositories associated with the content view (for the next publish), not the 'snapshot' of those repositories.  Those are stored on the content view version